tox.ini

```diff
@@ -33,7 +33,7 @@ setenv =
 
 # Run the tests in a temporary directory to make sure that we don't import
 # astropy from the source tree
-changedir = .tmp/{envname}
+changedir = $HOME/.local/.tmp/{envname}
```

Comment:
Is this a good idea? Any failure means one is just filling up a piece of one's home directory that one would be very unlikely to ever look at. Would it make sense to instead create this inside the `tox` environment? i.e, `changedir = .tox/{envname}` or `.tox/{envname}/tmp` or so?

Reply:
I tried `.tmp/tmp/{envname}` but it still failed. Would changing `.tmp` to `.tox` even help? I dunno.

Reply:
Yes, I guess I don't even understand why anything away from the astropy root fails in the first place...

Reply:
Didn't work anyway. I removed it.